src/fshtypes/Extension.ts

```diff
@@ -3,10 +3,13 @@ import { Rule } from './rules/Rule';
 export class Extension {
   id: string;
   parent?: string;
+  title?: string;
+  description?: string;
   rules: Rule[];
 
   constructor(public name: string) {
     this.id = name; // init same as name
+    this.parent = 'Extension'; // init to 'Extension'
```

Comment:
There is no obvious _default_ parent for a profile -- it could be any one of the 130 or so FHIR resource types.  But I'd say that 99.9% of extensions are based directly on `Extension` (and 100% must eventually be based on `Extension`).

I'll add a comment to make this more clear.